### PR TITLE
Feature 5

### PR DESCRIPTION
**Pull request by withlovedese**

### CODE REVIEW

**Comment by withlovedese** [comment]
Was trying to implement list format for comments

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -37,9 +40,16 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    //greets.add("Whats up");
-    //greets.add("Hi Dese!");
-    //greets.add("Hows it going");
+    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: You could name the entity "Comment" or equivalent to make it more relevant to our usecase

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -53,13 +63,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String text = getParameter(request, "text-input", "");
+    long timestamp = System.currentTimeMillis();
 
     //create entity
     Entity taskEntity = new Entity("Task");
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:69]: Could label it as commentEntity to make more relevant to our usecase

**Review by ReshmaMary-gcloud — APPROVED** [review]
Looks good

**Pull request merged**